src/sentry/migrations/0160_create_projectcodeowners_table.py

```diff
@@ -0,0 +1,76 @@
+# Generated by Django 1.11.29 on 2021-02-09 01:09
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+import sentry.db.models.fields.bounded
+import sentry.db.models.fields.foreignkey
+import sentry.db.models.fields.jsonfield
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    # You'll also usually want to set this to `False` if you're writing a data
+    # migration, since we don't want the entire migration to run in one long-running
+    # transaction.
+    atomic = True
+
+    dependencies = [
+        ("sentry", "0159_create_externaluser_table"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="ProjectCodeOwners",
+            fields=[
+                (
+                    "id",
+                    sentry.db.models.fields.bounded.BoundedBigAutoField(
+                        primary_key=True, serialize=False
+                    ),
+                ),
+                ("date_updated", models.DateTimeField(default=django.utils.timezone.now)),
+                ("date_added", models.DateTimeField(default=django.utils.timezone.now, null=True)),
```

Comment:
Why is this column nullable? It seems like `date_added` should be not null, and `date_updated` should be the nullable one.

Reply:
@markstory it's coming from https://github.com/getsentry/sentry/blob/master/src/sentry/db/models/base.py#L128
Associated PR: https://github.com/getsentry/sentry/pull/18535

Ahh it looks like it was setup this way for backfilling. But I think the default will automatically set the timestamp unless we explicitly set it as null? 

Is there a different base class you would recommend? Or should we keep it as is for consistency (we use it for all the `integrations`-related models)

Reply:
It's probably worth just overriding the field on your model - you can just copy the field from the base model and remove the `null=True`. We should probably look into removing this from the base class, since mostly new models are going to be using this field. We can set `null=True` on the models that require it instead.